src/core/src/op/strided_slice.cpp

```diff
@@ -237,27 +238,54 @@ bool op::v1::StridedSlice::has_evaluate() const {
     return get_input_size() == 4;
 }
 
-namespace {
-bool strided_slice_input_check(const ov::Node* node) {
-    if (!node->get_input_tensor(1).has_and_set_bound() || !node->get_input_tensor(2).has_and_set_bound() ||
-        !node->get_input_tensor(3).has_and_set_bound())
+bool op::v1::StridedSlice::indicies_input_has_and_set_bounds(const size_t port,
+                                                             const std::vector<int64_t>& masks) const {
+    const auto& lb_t = get_input_tensor(port).get_lower_value();
+    const auto& ub_t = get_input_tensor(port).get_upper_value();
+
+    if (lb_t && ub_t) {
```

Comment:
Good catch, just missed that case. Updated with solution and unit test for that case.

However I had to also modify the `constant_fold` function in case when there is Parameter{3} and bit mask are defined for all dimension it looks like this input must be replaced by dummy constant to correctly process bound evaluation. Wihtout it the `could_propagate` in the evaluation will return false and evaluates of `StridedSlice` will be not executed.

If required I making clone of op with replaced inputs and then try get it as constant.